src/main/java/org/ibp/api/java/impl/middleware/study/validator/StudyValidator.java

```diff
@@ -92,6 +92,23 @@ public void validateStudyContainsEntry(final Integer studyId, final Integer entr
 		}
 	}
 
+	public void validateStudyContainsEntries(final Integer studyId, final List<Integer> entryIds) {
```

Comment:
This should be consolidated with `validateStudyContainsEntry` (that is the usage of validateStudyContainsEntry should be replaced with this one) Also, move this validation to `StudyEntryValidator`.